### ISSUE TITLE
Don't colorize redirected debug output

### ISSUE THREAD

**Issue by philrz** [brimdata/zed#5237]
## tl;dr

When the `stderr` output of the `debug` operator is redirected to a file, its ZSON contents may be colorized. This seems undesirable.

## Details

Repro is with Zed commit 2e2c19b.

Start with this test data `conn.zson`:

```
{_path:"conn",ts:2018-03-24T17:15:21.255387Z,uid:"CMwJl820SlYyy434c8"}
```

If I render its contents as ZSON and redirect the main output to one file and the debug output on `stderr` to another, I get plain text in both files.

```
$ zq -z 'debug' conn.zson > main.out 2> debug.out

$ less main.out 
{_path:"conn",ts:2018-03-24T17:15:21.255387Z,uid:"CMwJl820SlYyy434c8"}

$ less debug.out 
{_path:"conn",ts:2018-03-24T17:15:21.255387Z,uid:"CMwJl820SlYyy434c8"}
```

However, if I let the main output flow to the screen (where it now appears colorized) while still redirecting the debug output on `stderr` to a file, now I get colorized output in the redirected debug output.

```
$ zq -z 'debug' conn.zson 2> debug.out
{_path:"conn",ts:2018-03-24T17:15:21.255387Z,uid:"CMwJl820SlYyy434c8"}

$ less debug.out 
"debug.out" may be a binary file.  See it anyway? y
{ESC[38;5;4m_pathESC[0m:ESC[38;5;2m"conn"ESC[0m,ESC[38;5;4mtsESC[0m:ESC[0m2018-03-24T17:15:21.255387ZESC[0m,ESC[38;5;4muidESC[0m:ESC[38;5;2m"CMwJl820SlYyy434c8"ESC[0m}
```

Since we're already clever enough to enable/disable colorization based on whether the main output is going to the terminal, hopefully we can do something similar to disable it when the debug output isn't going to a terminal, independent of where the main output is going. I imagine users would appreciate this since they may look at debug output in an editor or want to run it through `zq`, and having the color formatting in there impacts the ability to do both of those.
